docker-load-execute.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+set -e
+
+start_api() {
+    echo 'Starting API'
+    export DB_URL=postgresql
+    export DB_PORT=5432
+    export DB_NAME=intermediary-test
+    export DB_USER=intermediary
+    export DB_PASS=changeIT!
+    export DB_SSL=require
+    export REPORT_STREAM_URL_PREFIX=
+    ./gradlew shadowJar
+    docker compose up --build -d
+    docker compose logs router --no-color > docker-load-test-logs.txt
+    export API_PID="${!}"
+    echo "API starting at PID ${API_PID}"
+}
+
+start_database() {
+    echo 'Starting database'
+    docker compose -f docker-compose.postgres-test.yml up -d
+    sleep 2
+    echo "Database started"
+}
+migrate_database() {
+    echo 'Migrating database'
+    liquibase update --changelog-file ./etor/databaseMigrations/root.yml --url jdbc:postgresql://localhost:5434/intermediary-test --username intermediary --password 'changeIT!' --label-filter '!azure'
```

Comment:
Would it be worth extracting the port number into a variable and place that at the top of the file? Given the port may change more frequently? Also, this port and `DB_PORT` above are different. Why is that?